ui/tests/e2e/custom-playwright-reporter.js

```diff
@@ -6,39 +6,37 @@ import path from 'path';
 class MyReporter {
   introMessage = '';
   totalTests = '';
-  testData = [];
+  expectedTest = '';
+  testTable = `| Test | Browser | Test Case | Tags | Result |
+| :---: | :---: | :--- | :---: | :---: |`;
   passed = 0;
   failed = 0;
   skipped = 0;
   flaky = 0;
   countLog = 1;
-
-  constructor() {
-    this.countTestStatus = this.countTestStatus.bind(this);
-  }
+  countTable = 1;
 
   onBegin(_config, suite) {
     this.introMessage = `- Testing started at: ${moment().format('MMMM Do YYYY, h:mm:ss a')}`;
     this.totalTests = `- Total test cases: ${suite.allTests().length}`;
   }
-
-  onStdOut(chunk) {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
why here eslint warning need to be ignored? I f these vars not need why we are not removing them?

Reply:
This reverts the changes to the reporter that I previously made. This here prior to my changes. I can remove them again if you'd like. There will be a separate effort to enhance the reporting as we iterate and get visibility funcitionatly working